go.mod

```diff
@@ -21,5 +21,7 @@ require (
 	golang.org/x/arch v0.0.0-20190927153633-4e8777c89be4
 	golang.org/x/sys v0.0.0-20190626221950-04f50cda93cb
 	golang.org/x/tools v0.0.0-20191127201027-ecd32218bd7f
+	gopkg.in/airbrake/gobrake.v2 v2.0.9 // indirect
```

Comment:
Why do these dependencies keep showing up? As far as I can tell nothing is using them.

Reply:
I did not add them manually :) I took them out now. I think the previous merge must have not gone right, and they did not get in sync with master.